feature/staff/src/main/java/io/github/droidkaigi/confsched2020/staff/ui/StaffsFragment.kt

```diff
@@ -78,16 +76,12 @@ class StaffsFragment : Fragment(R.layout.fragment_staffs) {
             }
         }
 
-        val progressTimeLatch = ProgressTimeLatch { showProgress ->
-            binding.progressBar.isVisible = showProgress
-        }.apply {
-            loading = true
-        }
+        binding.progressBar.show()
 
         // This is the transition for the stagger effect.
         val stagger = Stagger()
         staffsViewModel.uiModel.observe(viewLifecycleOwner) { uiModel ->
-            progressTimeLatch.loading = uiModel.isLoading
+            with(binding.progressBar) { if (uiModel.isLoading) show() else hide() }
```

Comment:
How about creating an extension function? 🔄

Reply:
I think so nice! I create it! 🙌